### ISSUE TITLE
Can't install with pip

### ISSUE THREAD

**Issue by djmartz77** [luissilva1044894/Pyrez#1]
When trying to install Pyrez using pip, I got a syntax error. Here is the full text from the terminal session

> pip install -U pyrez
Collecting pyrez
  Downloading https://files.pythonhosted.org/packages/e9/0a/ebf2a68d24fde826132a33116eafcb9013308defae8aafdf2e0bb6fd030c/Pyrez-0.9.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-UQReGq/pyrez/setup.py", line 26
        def regexFunc (pattern: str, string: str = readFile ("pyrez/__init__.py")):
                              ^
    SyntaxError: invalid syntax
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-UQReGq/pyrez/

I'm running Ubuntu 18.04

**Comment by luissilva1044894** [comment]
I'll check it! I haven't got it. I think that it's a error about Regex (It's a error on setup.py). You have it already installed? I'm using Windows.

**Comment by djmartz77** [comment]
I didn't install Pyrez, actually. I'm still a Python beginner, so when pip didn't work I decided to take it as a challenge and write my own program to interface with the Hirez API. I know that's a bit silly, but it's been a very good exercise so far.

Maybe I misunderstood, though. Were you asking if I had installed Pyrez or something else?

**Comment by luissilva1044894** [comment]
Nop, was about Regex. I'm a Python beginner too, so idk all things about pip / setup.py. But I think that isn't necessary to use the API.

Whatever, I coded it to myself, but I know how interesting is it. So, free fell to use my repo as reference.  (It's why I uploaded it here!)

If you got stuck about something, maybe this can help you: [https://github.com/apugh/realm-api-docs/wiki](Realm API Wiki)
